### ISSUE TITLE
Sphinx requirement seems wrong

### ISSUE THREAD

**Issue by kayhayen** [dzhu/rstfmt#18]
Hello,

seems sphinx has become incompatible:

```
    rst_extras.register()
  File "/usr/local/lib/python3.9/site-packages/rstfmt/rst_extras.py", line 145, in register
    _add_directive("literalinclude", sphinx.directives.code.LiteralInclude)
  File "/usr/local/lib/python3.9/site-packages/sphinx/deprecation.py", line 48, in __getattr__
    return getattr(self._module, name)
AttributeError: module 'sphinx.directives' has no attribute 'code'
```

Downgrading sphinx to 2.4 solves this.

**Comment by dzhu** [comment]
Thanks for the report! This should be fixed in version 0.0.10, now available on PyPI.

**Comment by kayhayen** [comment]
Thanks a lot, workings for me
